transmart-rest-api-e2e/src/test/groovy/tests/rest/v2/SubscriptionSpec.groovy

```diff
@@ -0,0 +1,74 @@
+package tests.rest.v2
+
+import base.RESTSpec
+
+import static base.ContentTypeFor.JSON
+import static config.Config.*
+
+/**
+ * These tests tequires transmart-notifications plugin to be enabled and properly configured.
+ * See transmart-notifications configuration description.
+ */
+class SubscriptionSpec extends RESTSpec {
+
+    void 'test triggering email sending by a regular user is denied'() {
+        when: "I try to trigger email sending as default user"
+        def frequency = ""
+        def responseData = get([
+                path      : PATH_SUBSCRIPTION,
+                acceptType: JSON,
+                query     : [frequency: frequency],
+                user      : DEFAULT_USER,
+                statusCode: 403
+        ])
```

Comment:
You can now convert this to `ErrorResponse` so that we also have type checking in our test code.